client/src/components/exercise/lab9/pages/Repair.js

```diff
@@ -27,13 +46,14 @@ const Repair = (props) => {
         heading={headingText}
         justifyAlignment={"space-between"}
       />
-      <div className="tw-pb-10 tw-text-xl">
+      <div className="tw-pb-10 tw-text-xl ">
         {repairText.map((text, index) => (
           <p className="tw-indent-2" key={index}>
```

Comment:
nit: i commented in heather's pr the other day. let's move away from utilize index as key. not a big issue, but bad practice